xpallets/transaction-fee/rpc/src/lib.rs

```diff
@@ -68,10 +68,12 @@ where
 
         api.query_fee_details(&at, uxt, encoded_len)
             .map(|fee_details| FeeDetails {
-                inclusion_fee: fee_details.inclusion_fee.map(|fee| InclusionFee {
-                    base_fee: fee.base_fee.into(),
-                    len_fee: fee.len_fee.into(),
-                    adjusted_weight_fee: fee.adjusted_weight_fee.into(),
+                inclusion_fee: fee_details.inclusion_fee.map(|fee| {
+                    pallet_transaction_payment::InclusionFee {
+                        base_fee: fee.base_fee.into(),
+                        len_fee: fee.len_fee.into(),
+                        adjusted_weight_fee: fee.adjusted_weight_fee.into(),
+                    }
```

Comment:
```suggestion
                inclusion_fee: fee_details.inclusion_fee.map(|fee| InclusionFee {
                    base_fee: fee.base_fee.into(),
                    len_fee: fee.len_fee.into(),
                    adjusted_weight_fee: fee.adjusted_weight_fee.into()
                }),
```

Reply:
If you import `pallet_transaction_payment::InclusionFee` in the header, you don't have to change it here (I think it would be better)